src/module/module.py

```diff
@@ -18,7 +18,8 @@
 
 # connect the client to a remote MQTT broker
 log.debug(f'Connecting to MQTT... Broker: {PARAMS["MQTT_BROKER"]} Port: {PARAMS["PORT"]}')
-client.connect(host=PARAMS['MQTT_BROKER'], port=PARAMS['PORT'])
+__MQTT_BROKER__ = PARAMS['MQTT_BROKER'] if not 'mqtt://' in PARAMS['MQTT_BROKER'] else PARAMS['MQTT_BROKER'].replace('mqtt://', '')
```

Comment:
what if the user inputs `mqtts://...` or `tls://...`?

Reply:
The package accepts only `mqtt://` so if there is `mqtts://` or `tls://` then the next line that connects to the broker will throw the error and not connect.

Reply:
you are right. but I just tested it and the error is not very eloquent `socket.gaierror: [Errno -2] Name or service not known`. I assume the user would not understand right away that the protocol is not supported.
let's check the broker url for `://` and remove the scheme if it's detected

Reply:
Addressed it